### ISSUE TITLE
The date in investment editor does not use the date from the picker

### ISSUE THREAD

**Issue by MisterY** [moneymanagerex/android-money-manager-ex#890]
Discovered in #889. 
The transaction date is not taken from the date picker. Check if there is a difference between new/existing record saving.
The same behaviour for moving dates back/forth should be added after the correction.
Ping @vladimir-poleh

**Comment by vladimir-poleh** [comment]
There is no easy way to edit Investment. I have found only this: from the context menu for the Investment account on the Home screen select "Portfolio". Clicking item in the portfolio will open edit dialog.
Clicking items in the Investment account transactions list will show menu with "Download price", "Edit price" and "Delete" options. I think that it will be good to add "Edit" to this context menu.

**Comment by MisterY** [comment]
That's correct. The process needs to be a bit improved and made more obvious, I agree. The current state is the result of, on one side, waiting for the desktop team to come up with the data schema so that I could implement the solution on Android, and on another the need to have some functional solution right now.
Actual investment transactions are edited in Portfolio. That form edits individual records.
The other view was created first, and is intended to serve as a watchlist. It lists the symbols only, with their last known prices. Hence it makes no sense to add Edit there, because there is no (or should not be) record to edit. Downloading prices simply updates the last known price on an investment record. Watchlist was intended for a quick price update and export in CSV format, which can be imported into Quicken or elsewhere.
I think that there should be a clear distinction between a Portfolio and a Watchlist. The default view, when clicking an Investment account, should probably be Portfolio. 

In conclusion, the edit form that appears when tapping a record in Portfolio contains a date picker that would be equivalent to the forms modified so far.
Ina  Watchlist there is an option to manually update a price (which I also actively use for the funds that do not expose any API for price download) and there would also be extremely convenient to be able to quickly move up and down, when setting the price for yesterday, for example.
